README.md

```diff
@@ -38,13 +38,40 @@ java -jar aws-smtp-relay.jar
 ### Arguments
 ```
 usage: aws-smtp-relay
- -b,--bindAddress <arg>     Address to listen to
- -c,--configuration <arg>   AWS SES configuration to use
- -a,--sourceArn <arg>       AWS ARN of the sending authorization policy
- -h,--help                  Display this help
- -p,--port <arg>            Port number to listen to
- -r,--region <arg>          AWS region to use
+ -a,--sourceArn <arg>           AWS ARN of the sending authorization policy
+ -b,--bindAddress <arg>         Address to listen to
+ -c,--configuration <arg>       AWS SES configuration to use
+ -h,--help                      Display this help
+ -p,--port <arg>                Port number to listen to
+ -r,--region <arg>              AWS region to use
+ -smtpH,--smtpHost <arg>        SMTP variable Host
```

Comment:
I would use ssmHost for SMTP variable Host and so on ssmPort instead of smtpPort as all of them are related to SSM.

Reply:
This can be configured with or without ssm. It was built to be fully cli'ed but if ssm is available then go use the prefix to find the same variables.

After using the service for quite a while. I've found the smtpOverride variable is not informative when looking over it. 

smtpOverride = f (ses is enabled
smtpOverride = t (ses is disabled, use smtp settings provided)

I thinking of changing it to sesDisabled = true or something similar as this module is about shunting emails directly to SES more so than smtp mailing.

Reply:
I just got it now working more in details to merge your PR, i thought it was to specify the SSM variable not the SMTP settings provided

Reply:
Yes its confusing as its two improvements in one.
The Idea is to allow you to setup a dummy smtp credentials.
Then we bolted on SSM since we have gone fully in on this to check if its good.
You could build in a last checked configuration info and reload which smtp provider is being run but then you need to ensure it does not have a memory leak in doing so or allow itself to be rebooted if it does see a change.